week-1-sean-wallace.js

```diff
@@ -0,0 +1,208 @@
+
+const currentCohort = "Delta 2022"
+let string = ("Delta 2022")
+string = [...string].reverse().join("");
+
+console.log(string);
+
+
+
+
+// --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
+
+// Pseudo code: Use for, if, else, for let index = 0; then i < myArray and .length, i++ for the value before the increment. if myArray index modulo by 2 strict inequal the 0 console.log odd else if myArray i console.log even.
+
+const myArray = [13, 34, 5, 10, 27, 42]
+for(let i = 0; i < myArray.length; i++) 
+   if(myArray[i] % 2 !== 0) {
+      console.log(`odd`)  
+    }
+    else if(myArray[i]) {
+        console.log(`even`)
+      }
+    
+
+
+
+
+
+// --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
+
+// Pseudo code: console.log number2-number1; console.log number3-number4
+
+// Set one:
+
+const number1 = 58
+const number2 = 100
+console.log(number2-number1)
+
+// Set two:
+const number3 = 27
+const number4 = 24
+console.log(number3-number4)
+
+// ASSESSMENT 1: Coding Conceptual Questions
+
+// Examine the following examples.
+
+// First, predict the outcome based on your understanding of the code.
+// Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
+
+// --------------------INSTRUCTOR EXAMPLE: What will this log?
+const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+//console.log(colors.push("indigo"))
+
+// a) Your answer: will push indigo to the end
+// b) Verify and explain: .push places any variable to the end of a string.
+
+
+// --------------------1) What will this log?
+
+const cohort = "LEARN 2022"
+//console.log(cohort.length)
+
+// a) Your answer: 10
+// b) Verify and explain: the .length counts the number of digits in the string.
+
+
+// --------------------2) What will this log?
+
+const greeting = "Hello World!"
+console.log(greeting[4])
+
+// a) Your answer:o
+// b) Verify and explain: the 4 counts the letters in the string.
+
+
+// --------------------3) What will this log?
+
+const languages = ["JavaScript", "Ruby", "Python", "C++"]
+const index = 1
+console.log(languages[index])
+
+// a) Your answer: Ruby
+// b) Verify and explain:  because it string starts at 0 ruby is in 1.
+
+
+// --------------------4) What will this log?
+
+const weekendDays = ["saturday", "sunday"]
+console.log(weekendDays.toUpperCase())
+
+// a) Your answer: capitalize all caps in weekendDays
+// b) Verify and explain: not gettin a output. but i think it will uppercase all in the weekendDays because of the .toUpperCase console.log.
```

Comment:
For sure, this should throw an error because .toUpperCase() only works on strings and not on arrays.